zkclient/net.go

```diff
@@ -40,18 +40,8 @@ func (mw *mainWindow) kx(conn io.ReadWriteCloser) error {
 	// read remote public identity
 	var remoteId zkidentity.PublicIdentity
 	_, err = xdr.Unmarshal(conn, &remoteId)
-	if err != nil {
-		return err
-	}
 
-	// handle trust
-	//err = handleTrust(remoteId)
-	//if err != nil {
-	//	// printing has happened
-	//	return err
-	//}
```

Comment:
i removed it.. the function doesn't exist.  perhaps i should leave a single line
```// TODO handle trust```  ?

or make an issue on GH?